### ISSUE TITLE
SALT is broken

### ISSUE THREAD

**Issue by gfx** [bitjourney/plantuml-service#26]
given the following source:

~~~plantuml
@startuml
salt
{
{T
 + World
 ++ America
 +++ Canada
 +++ USA
 ++++ New York
 ++++ Boston
 +++ Mexico
 ++ Europe
 +++ Italy
 +++ Germany
 ++++ Berlin
 ++ Africa
}
}
@enduml
~~~

expected (using www.plantuml.com):

[![](http://www.plantuml.com/plantuml/svg/DOv12y8m38Nl_HMvzqzioX0FUb4HZs6DK4mRIJjqZFrtjtRoBbyUITvhKaRDGs2JaBEPp7GrOE4kocu12sqWzJskR66F4Tt6jqjRmCAPFl0GVMtZJbAMM9TEzFMzb4VTeFAcQXypybZnG1emZlzRKlQn1ZzBxhoeeUZMYZy0)](http://www.plantuml.com/plantuml/svg/DOv12y8m38Nl_HMvzqzioX0FUb4HZs6DK4mRIJjqZFrtjtRoBbyUITvhKaRDGs2JaBEPp7GrOE4kocu12sqWzJskR66F4Tt6jqjRmCAPFl0GVMtZJbAMM9TEzFMzb4VTeFAcQXypybZnG1emZlzRKlQn1ZzBxhoeeUZMYZy0)

as of plantml-service 1.2.3 (using plantuml-service.herokuapp.com):

<a href="https://plantuml-service.herokuapp.com/svg/%40startuml%0D%0Asalt%0D%0A%7B%0D%0A%7BT%0D%0A%20%2B%20World%0D%0A%20%2B%2B%20America%0D%0A%20%2B%2B%2B%20Canada%0D%0A%20%2B%2B%2B%20USA%0D%0A%20%2B%2B%2B%2B%20New%20York%0D%0A%20%2B%2B%2B%2B%20Boston%0D%0A%20%2B%2B%2B%20Mexico%0D%0A%20%2B%2B%20Europe%0D%0A%20%2B%2B%2B%20Italy%0D%0A%20%2B%2B%2B%20Germany%0D%0A%20%2B%2B%2B%2B%20Berlin%0D%0A%20%2B%2B%20Africa%0D%0A%7D%0D%0A%7D%0D%0A%40enduml%0D%0A">
<img src="https://plantuml-service.herokuapp.com/svg/%40startuml%0D%0Asalt%0D%0A%7B%0D%0A%7BT%0D%0A%20%2B%20World%0D%0A%20%2B%2B%20America%0D%0A%20%2B%2B%2B%20Canada%0D%0A%20%2B%2B%2B%20USA%0D%0A%20%2B%2B%2B%2B%20New%20York%0D%0A%20%2B%2B%2B%2B%20Boston%0D%0A%20%2B%2B%2B%20Mexico%0D%0A%20%2B%2B%20Europe%0D%0A%20%2B%2B%2B%20Italy%0D%0A%20%2B%2B%2B%20Germany%0D%0A%20%2B%2B%2B%2B%20Berlin%0D%0A%20%2B%2B%20Africa%0D%0A%7D%0D%0A%7D%0D%0A%40enduml%0D%0A"/></a>

**Comment by gfx** [comment]
It is not fixed in #30. Reopened.

**Comment by gfx** [comment]
Gotcha! `+` is replaced into ` ` by `URLDecoder.decode()`!
